### PR TITLE
Update EDS getting started

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR updates Get Started with Event Delivery.

### CODE REVIEW

File: docs/photoniq/event-delivery/get-started-event-delivery.md [hunk 1]
```diff
@@ -9,42 +9,40 @@ However, some people like to be more hands-on with their management. This page p
...
 
-### Local or Global Stream
+Event Delivery lets you filter events from Macrometa streams and publish events to streams, so the first step is to get your data into a collection stream in the Macrometa Global Data Network (GDN).
 
 A stream is a named channel for sending messages. Each stream is backed by a distributed append-only log and can be local (at one edge location only) or global (across all edge locations in the [fabric](../../geofabrics/)).
 
-For more information, refer to [Streams](../../streams/).
-
-### Collection Stream
+1. (Optional) [Create a fabric](../../geofabrics/create-geofabric.md) for your collection. This allows you to decide exactly where your data lives. If you skip this step, then the collection will probably be in the system fabric.
```
Comment (manoj-macrometa) [inline docs/photoniq/event-delivery/get-started-event-delivery.md]: I think we should remove the probably word from this sentence.

File: docs/photoniq/event-delivery/get-started-event-delivery.md [hunk 2]
```diff
@@ -9,42 +9,40 @@ However, some people like to be more hands-on with their management. This page p
...
 
 ## Set Up Event Delivery
 
 Once your stream is set up in the GDN, you can subscribe to it with Event Delivery and manage it using the [Event Delivery API](https://www.macrometa.com/docs/apiEds#/).
 
-1. [Subscribe to the stream](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get). This includes adding any [Event Delivery Queries](event-delivery-queries.md) to filter events.
-2. [Publish events](https://www.macrometa.com/docs/apiEds#/paths/api-es-v1-fabric-fabricName--stream--streamName--publish/post). You can use this API endpoint or use one of the options for publishing events to the GDN stream.
+1. [Subscribe to the stream](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get). This includes adding any [Event Delivery queries](event-delivery-queries.md) to filter events.
+2. [Publish events](https://www.macrometa.com/docs/apiEds#/paths/api-es-v1-fabric-fabricName--stream--streamName--publish/post). You can use this API endpoint or use one of the options for publishing events to the GDN stream. Events published in this matter are not stored in the collection.
```
Comment (manoj-macrometa) [inline docs/photoniq/event-delivery/get-started-event-delivery.md:45]: we could also mention how a GDN stream can be created like how its is mentioned for collection above.

Comment (diana-macrometa) [reply docs/photoniq/event-delivery/get-started-event-delivery.md:45]: I'm not sure what you mean by that.

If you mean the GDN Streams feature, Durga wanted this topic simplified to just collection streams, so I took out the content talking about GDN streams.

File: docs/photoniq/event-delivery/get-started-event-delivery.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/event-delivery/get-started-event-delivery.md]: Removed, it just seems inaccurate. I mean, what if someone else on the account made a fabric and switched to it?

**Review by manoj-macrometa — APPROVED** [review]

**Pull request merged**